.github/workflows/pkgdown.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
unrelated, but they have a newer version of the script available. We're using it in the other action already.